app/forms.py

```diff
@@ -16,17 +17,22 @@ class RegistrationForm(FlaskForm):
     submit = SubmitField('Sign up')
 
     @staticmethod
-    def validate_army_name(army_name):
+    def validate_army_name(self, army_name):
         user = User.query.filter_by(army_name=army_name.data).first()
         if user:
             raise ValidationError('This army name is already taken')
 
     @staticmethod
-    def validate_email(email):
+    def validate_email(self, email):

```

Comment:
Why do you use `self` if it's a static method?

Reply:
i will change this method type, my bad